src/modules/iasql_functions/rpcs/iasql_install.ts

```diff
@@ -3,8 +3,34 @@ import { TypeormWrapper } from '../../../services/typeorm';
 import { Context, RpcBase, RpcResponseObject } from '../../interfaces';
 import * as iasql from '../iasql';
 
+/**
+ * Method to install the IaSQL modules provided by the engine
+ *
+ * Returns following columns:
+ * - module_name: Name of the module that was installed
+ * - created_table_name: Name of the associated table that was created
+ * - record_count: Total of registers added
+ *
+ * Accepts the following parameters:
+ * - list of modules to install
+ *
+ * @example
+ * ```sql
+ * SELECT * FROM iasql_install('aws_account');
```

Comment:
the user doesn't typically call this since it is done by the dashboard so let's use another module like vpc or ec2?